deploy/examples/create-external-cluster-resources.py

```diff
@@ -197,6 +196,8 @@ def gen_arg_parser(cls, args_to_parse=None):
                                   help="Provides the name of erasure coded RBD metadata pool")
         output_group.add_argument("--dry-run", default=False, action='store_true',
                                   help="Dry run prints the executed commands without running them")
+        output_group.add_argument("--rados-namespace",default="", required=False,
```

Comment:
Where is the code that verifies the namespace exists?

Reply:
ohh didn't think about this, 
yes, the logic that checks the --rados-namespace passed by the user exists or is not in the pool.